utmap-server/src/app/main/config.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+class Config:
+    SECRET_KEY = os.getenv('SECRET KEY', 'skeleton_key_sans_undertale_haha')
```

Comment:
`* You feel like you're going to have a bad time`

Reply:
Lol